### PR TITLE
Only reject a message if the answer count is > 1

### PR DESCRIPTION
**Pull request by paulf69487623**
See issue #898 for full details.

### CODE REVIEW

File: acceptfunc.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func defaultMsgAcceptFunc(dh Header) MsgAcceptAction {
 	if dh.Qdcount != 1 {
 		return MsgReject
 	}
-	if dh.Ancount != 0 {
+	if dh.Ancount > 1 {
```
Comment (tmthrgd) [inline acceptfunc.go]: Could you update the `DefaultMsgAcceptFunc` docs to reflect this and #868?

**Comment by miekg** [comment]
you can reference RFC 1996:

"Notify requests have one SOA in the Answer section. See RFC 1996 Section 3.7 and 3.11."

**Comment by miekg** [comment]
morphed this in #900. Also added acceptfunc_test.go the triggered the reject and works after this fix is merged.
Doc are updated there as well.

Kept original attribution

**Comment by miekg** [comment]
closing here. I'll merge #900 when travis is green.

**Pull request closed without merging**